invenio/templates/deployments/routing.yaml

```diff
@@ -15,6 +15,7 @@ spec:
     insecureEdgeTerminationPolicy: Redirect
     termination: edge
 {{- else }}
+{{- if .Values.ingress.enabled }}
```

Comment:
Use case is: our new K8s cluster uses the Istio ingress instead of nginx-ingress. So I think it's better to make the ingress excludable until there is a better solution (perhaps defining it in values.yaml and using toYaml to include it in the chart). 
As OpenShift doesn't have the needs for an Ingress (there is Route instead) I thought the if is better placed around the Kubernetes part.